scalardb-analytics-postgresql-sample/sample-data-loader/src/main/java/sample/data/Loader.java

```diff
@@ -0,0 +1,183 @@
+        .textValue("c_address", stringCol(record, "c_address"))
+        .intValue("c_nationkey", intCol(record, "c_nationkey"))
+        .textValue("c_phone", stringCol(record, "c_phone"))
+        .doubleValue("c_acctbal", doubleCol(record, "c_acctbal"))
+        .textValue("c_mktsegment", stringCol(record, "c_mktsegment"))
+        .textValue("c_comment", stringCol(record, "c_comment"))
+        .build();
+  }
+
+  private Put buildPutOrders(CSVRecord record) {
+    return Put.newBuilder()
+      .namespace("postgresns")
+      .table("orders")
+      .partitionKey(Key.ofInt("o_orderkey", intCol(record, "o_orderkey")))
+      .intValue("o_custkey", intCol(record, "o_custkey"))
+      .textValue("o_orderstatus", stringCol(record, "o_orderstatus"))
+      .doubleValue("o_totalprice", doubleCol(record, "o_totalprice"))
+      .textValue("o_orderdate", stringCol(record, "o_orderdate"))
+      .textValue("o_orderpriority", stringCol(record, "o_orderpriority"))
+      .textValue("o_clerk", stringCol(record, "o_clerk"))
+      .intValue("o_shippriority", intCol(record, "o_shippriority"))
+      .textValue("o_comment", stringCol(record, "o_comment"))
+      .build();
+  }
+
+  private Put buildPutLineitem(CSVRecord record) {
+    return Put.newBuilder()
+      .namespace("cassandrans")
+      .table("lineitem")
+      .partitionKey(Key.of("l_orderkey", intCol(record, "l_orderkey"), "l_linenumber", intCol(record, "l_linenumber")))
+      .intValue("l_partkey", intCol(record, "l_partkey"))
+      .intValue("l_suppkey", intCol(record, "l_suppkey"))
+      .intValue("l_quantity", intCol(record, "l_quantity"))
+      .doubleValue("l_extendedprice", doubleCol(record, "l_extendedprice"))
+      .doubleValue("l_discount", doubleCol(record, "l_discount"))
+      .doubleValue("l_tax", doubleCol(record, "l_tax"))
+      .textValue("l_returnflag", stringCol(record, "l_returnflag"))
+      .textValue("l_linestatus", stringCol(record, "l_linestatus"))
+      .textValue("l_shipdate", stringCol(record, "l_shipdate"))
+      .textValue("l_commitdate", stringCol(record, "l_commitdate"))
+      .textValue("l_receiptdate", stringCol(record, "l_receiptdate"))
+      .textValue("l_shipinstruct", stringCol(record, "l_shipinstruct"))
+      .textValue("l_shipmode", stringCol(record, "l_shipmode"))
+      .textValue("l_comment", stringCol(record, "l_comment"))
+      .build();
+  }
+
+  private void loadData(DistributedTransactionManager manager, String path, String[] columnHeader, Function<CSVRecord, Put> putFunction)
+    throws TransactionException, IOException {
+    DistributedTransaction transaction = null;
+    try (BufferedReader reader = Files.newBufferedReader(Path.of(path))) {
+      Iterable<CSVRecord> records = CSVFormat.Builder.create().setHeader(columnHeader).build().parse(reader);
+      transaction = manager.start();
+      for (CSVRecord record : records) {
+        Put put = putFunction.apply(record);
+        transaction.put(put);
+      }
```

Comment:
Ah, I see. I understand. Thank you for your answer!

> BTW, it would be more helpful if `mutate` could also accept `Iterable` in addition to `List` so that users can avoid loading all data into memory as `List`.

Sounds good! Let's discuss it in another place as an enhancement request!